easybuild/easyconfigs/l/libcifpp/libcifpp-7.0.8-GCC-13.3.0.eb

```diff
@@ -18,18 +18,34 @@ builddependencies = [
     ('CMake', '3.29.3'),
     ('Eigen', '3.4.0'),
     ('Boost', '1.85.0'),  # req. for tests
+    ('mrc', '1.3.13'),
+    ('Catch2', '2.13.10')
 ]
 
 dependencies = [
     ('zlib', '1.3.1'),
-    ('mrc', '1.3.13'),
 ]
 
+# CIFPP_DOWNLOAD_CCD is set to OFF in order to allow for consistent module rebuild.
+# Therefore, ciffpp data shipped with the src is installed into %(builddir)s/shared/libcifpp without an 
+# online update at build time.
+# However, "The files in CIFPP_DATA_DIR are quickly becoming out of date". The data can be installed elsewhere
+# outside the module directory in order to regularly run the update script - in this case
+# the variable CIFFPP_DATA_DIR and LIBCIFPP_DATA_DIR require to be adapted accordingly.
+# More information about the update script can be found here:
+# https://github.com/PDB-REDO/libcifpp#building
```

Comment:
Ok, are CIFFPP_DATA_DIR and LIBCIFPP_DATA_DIR configure time variables?
Then perhaps we can patch the code to take the data dir from an env variable and create a Dataset based easyconfig to handle the installation of the data.

For now this will do, but it might be worth considering to make a libcifpp-data easyconfig...

Reply:
I should have read further down. They are already env vars so it should be fairly easy to make a libcifpp-data package to handle this.
Is the data versioned already or would one just do yyyymmdd versioning of it?

Reply:
I'd suggest to use the data shipped with the sources by default. I think it would not make much sense to have a separate data module in this particular case, because [https://github.com/PDB-REDO/libcifpp#building](https://github.com/PDB-REDO/libcifpp#building) suggest to run a cron job for updates  on a weekly basis.